docs/Writing_yeb_easyconfig_files.rst

```diff
@@ -0,0 +1,359 @@
+.. _easyconfig_yeb_format:
+
+Writing easyconfig files in YAML syntax (``.yeb`` format) **[IN DEVELOPMENT]**
+==============================================================================
+
+.. note::
+    Because support for easyconfig files in YAML syntax (a.k.a. ``.yeb`` files) is still *in development*,
+    using them currently requires enabling the use of experimental features (``--experimental``),
+    see also :ref:`experimental_features` .
+
+    An up-to-date overview of current progress on support for ``.yeb`` easyconfigs is available at
+    https://github.com/hpcugent/easybuild-framework/issues/1407.
+
+Useful links:
+
+* YAML syntax specification: http://www.yaml.org/spec/1.2/spec.html
+
```

Comment:
Is PyYAML a 1.2 parser? They advertise as 1.1 (though I imagine there are some extensions)